herald/src/data/generate-guild-data.js

```diff
@@ -0,0 +1,72 @@
+"use strict"
```

Comment:
Can you update this script to generate (at least) 1 guild with a static name, and then update `herald/src/app/+home/home.component.html` with a hyperlink to it?  This'll make it easier for people to get a quick glance at a profile without trying to figure out a dynamic guild name.